app/javascript/components/assets/mockUsers.js

```diff
@@ -0,0 +1,9 @@
+let users = [
```

Comment:
Usually, when mocking data, we want to make sure our objects match the structure of our schema.